xdsl/frontend/type_conversion.py

```diff
@@ -72,7 +72,7 @@ def _convert_name(self, type_hint: ast.Name) -> Attribute:
         # First, type can be generic, e.g. `class _Integer(Generic[_W, _S])`.
         if isinstance(type_class, _GenericAlias):
             generic_type_arguments = type_class.__args__
-            arguments_for_constructor = []
+            arguments_for_constructor = list[Any]()
```

Comment:
Can we not have `x: List[Any] = []`?

Reply:
Sure they're equivalent, I'm not sure which I prefer...

Reply:
Changed it to the other way